Intersect.Server/Entities/Player.cs

```diff
@@ -3149,10 +3149,14 @@ public bool CheckCrafting(Guid id)
         }
 
         //Business
-        public bool IsBusy()
-        {
-            return InShop != null || InBank || CraftingTableId != Guid.Empty || Trading.Counterparty != null;
-        }
+        private bool IsBusy() =>
```

Comment:
Hmm, I wonder if adding on to this method will cause problems for the _other_ places that use it? I'm not sure. But, pre-change, it _appears_ that a player would still be able to attack and move around when they have an invite open, which might actually be desired, as otherwise someone could spam them with invites to prevent a player from attacking them, for example.

We might need like, an `CanAcceptRequest` function instead that _only_ checks for requesters :P sorry for leading you astray